example/mosmix_forecasts.py

```diff
@@ -21,8 +21,8 @@ def mosmix_example():
     mosmix = DWDMosmixData(
         station_ids=['01001', '01008'],
         parameters=['DD', 'ww'],
-        start_date=ForecastDate.LATEST,  # automatically set if left empty
-        period_type=PeriodType.FORECAST_LONG,
+        start_date=DWDFcstDate.LATEST,  # automatically set if left empty
+        forecast_type=PeriodType.FORECAST_LONG,
```

Comment:
Is it really `PeriodType.FORECAST_LONG` instead of `DWDFcstPeriodType.FORECAST_LONG` here? Again, that might become `DWDMosmixPeriod.LONG`.